android/modules/geolocation/src/java/ti/modules/titanium/geolocation/GeolocationModule.java

```diff
@@ -381,21 +377,6 @@ private void propertyChangedFrequency(Object newValue)
 		double frequencyProperty = TiConvert.toDouble(newValue) * 1000;
 	}
 
-	/**
-	 * Handles property change for Ti.Geolocation.preferredProvider
-	 *
-	 * @param newValue					new preferredProvider value
-	 */
-	@SuppressLint("MissingPermission")
-	private void propertyChangedPreferredProvider(Object newValue)
```

Comment:
Looks like this was basically a no-op, so I can see why it's being removed - but our docs say deprecated since 2.0 with no removal..

Reply:
I just noticed the `Ti.Geolocation.frequency` property (deprecated in 2.0.0) is dead code too. I'll clean this up.

Reply:
I've updated the docs. Confirmed removed as of Titanium 8.0.0.